src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.test.js

```diff
@@ -1,10 +1,86 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { render } from '@testing-library/react';
 
-// import ValueList from 'index';
+import { string2date, string2time } from 'shared/services/string-parser/string-parser';
+import { withAppContext } from 'test/utils';
 
-describe('<ValueList />', () => {
-  it('Expect to have unit tests specified', () => {
-    expect(true).toEqual(true);
+import MetaList from './index';
+
+jest.mock('../../components/Highlight', () => () => <div data-testid="meta-list-highlight" />);
```

Comment:
By mocking this component, there's no way to verify that 'Urgentie' is shown on the page.

Reply:
that is true, but it does check for the existence of 5 highlight instances in the component:
expect(queryAllByTestId('meta-list-highlight')).toHaveLength(5);
I think that is fine.